src/main/scala/edu/ie3/simona/agent/ValueStore.scala

```diff
@@ -48,6 +48,15 @@ final case class ValueStore[+D](
       .filter(entry => entry._1 <= requestedTick)
       .maxByOption(entry => entry._1)
 
+  /** Get the last known entry (with the highest tick)
+    *
+    * @return
+    *   An Option to the last entry
+    */
+  def last(): Option[(Long, D)] =
+    store
+      .maxByOption(entry => entry._1)
```

Comment:
I'd rather like 
```suggestion
      .maxByOption(_._1)
```

or more expressive
```suggestion
      .maxByOption{
        case (tick, _) => tick
      }
```